web/src/main/java/org/cbioportal/web/util/DataBinner.java

```diff
@@ -41,17 +41,17 @@ public DataBinner(DataBinHelper dataBinHelper,
     }
     
     public List<DataBin> calculateClinicalDataBins(ClinicalDataBinFilter clinicalDataBinFilter,
+                                                   ClinicalDataType clinicalDataType,
```

Comment:
If the clinicalDataBinFilter already has ClinicalDataType, why not use that?

Reply:
clinicalDataBinFilter doesn't have ClinicalDataType

Reply:
Just saw in the previous implementation, there is `clinicalDataBinFilter.getClinicalDataType()`

Reply:
its removed https://github.com/cBioPortal/cbioportal/pull/6585/files#diff-e8c2c9fcef3054c28c70b5134669bc22L20

Reply:
does not seem being removed. 
```
            DataBin naDataBin = calcNaDataBin(attributeId, clinicalData, clinicalDataBinFilter.getClinicalDataType() ,ids);
```

Reply:
@khzhu I don't see it. I see
```DataBin naDataBin = calcNaDataBin(attributeId, clinicalData, clinicalDataType ,ids);```

Reply:
great, if it was removed.